testing/e2e/workload_test.go

```diff
@@ -516,6 +517,94 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 				return nil
 			},
 		},
+		{
+			Name:         "Update the created workload with `replace` update strategy",
+			WorkloadName: "spring-petclinic",
+			Command: func() it.CommandLine {
+				c := *it.NewTanzuAppsCommandLine(
+					"workload", "apply", "spring-petclinic",
```

Comment:
Do we need a test case with no update strategy specified so that it picks up the default and do nothing or show no change in workload?